x-pack/plugins/security_solution/server/lib/siem_migrations/rules/data/rule_migrations_data_rules_client.ts

```diff
@@ -5,70 +5,66 @@
  * 2.0.
  */
 
-import type { AuthenticatedUser, ElasticsearchClient, Logger } from '@kbn/core/server';
-import assert from 'assert';
 import type {
   AggregationsFilterAggregate,
   AggregationsMaxAggregate,
   AggregationsStringTermsAggregate,
   AggregationsStringTermsBucket,
   QueryDslQueryContainer,
-  SearchHit,
-  SearchResponse,
 } from '@elastic/elasticsearch/lib/api/types';
 import type { StoredRuleMigration } from '../types';
 import { SiemMigrationStatus } from '../../../../../common/siem_migrations/constants';
 import type {
   RuleMigration,
   RuleMigrationTaskStats,
 } from '../../../../../common/siem_migrations/model/rule_migration.gen';
+import { RuleMigrationsDataBaseClient } from './rule_migrations_data_base_client';
 
 export type CreateRuleMigrationInput = Omit<RuleMigration, '@timestamp' | 'status' | 'created_by'>;
 export type RuleMigrationDataStats = Omit<RuleMigrationTaskStats, 'status'>;
 export type RuleMigrationAllDataStats = Array<RuleMigrationDataStats & { migration_id: string }>;
 
-export class RuleMigrationsDataClient {
-  constructor(
-    private dataStreamNamePromise: Promise<string>,
-    private currentUser: AuthenticatedUser,
-    private esClient: ElasticsearchClient,
-    private logger: Logger
-  ) {}
+const BULK_MAX_SIZE = 500 as const;
```

Comment:
Same thing here for why this size was chosen, and maybe there should be a shared const if the values will remain consistently the same?